### PR TITLE
YONE 6 cards

### PR DESCRIPTION
**Pull request by Simisays**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/ichor_aberration.txt [hunk 1]
```diff
@@ -0,0 +1,17 @@
+Name:Ichor Aberration
+ManaCost:1 U B
+Types:Creature Phyrexian Horror
+PT:3/3
+K:Flying
+K:Defender
+S:Mode$ CanAttackDefender | ValidCard$ Card.Self+powerGE7 | Description$ As long as CARDNAME's power is 7 or greater, it can attack as though it didn't have defender. 
+T:Mode$ Proliferate | ValidPlayer$ You | TriggerZones$ Battlefield | TriggerZones$ Battlefield | IsPresent$ Card.Self | PresentZone$ Battlefield | Execute$ TrigPerpetualBuff | TriggerDescription$ Whenever you proliferate, if CARDNAME is in your graveyard or on the battlefield, CARDNAME perpetually gets +1/+1. You may cast it from your graveyard this turn.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/ichor_aberration.txt:8]: One trigger should be better even if you need SVar then

File: forge-gui/res/cardsfolder/upcoming/kembas_outfitter.txt [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+SVar:TrigCharm:DB$ Charm | Choices$ ReduceHand,ReduceBattlefield
+SVar:ReduceHand:DB$ ChooseCard | Mandatory$ True | ChoiceZone$ Hand | Choices$ Equipment.YouOwn | ChoiceTitle$ Choose an equipment card in your hand | Amount$ 1 | SubAbility$ DBEffect | SpellDescription$ Choose an equipment card in your hand. It perpetually gains equip {1}.
+SVar:DBEffect:DB$ Effect | RememberObjects$ ChosenCard | StaticAbilities$ PerpetualAbility | Duration$ Permanent | Name$ Kemba's Outfitter's Perpetual Effect
+SVar:ReduceBattlefield:DB$ Effect | ValidTgts$ Equipment.YouCtrl | RememberObjects$ Targeted | StaticAbilities$ PerpetualAbility | Duration$ Permanent | Name$ Kemba's Outfitter's Perpetual Effect | SpellDescription$ Target Equipment you control perpetually gains equip {1}.
+SVar:PerpetualAbility:Mode$ Continuous | Affected$ Card.IsRemembered | AddKeyword$ Equip:1 | EffectZone$ Command | AffectedZone$ Battlefield,Hand,Graveyard,Exile,Stack,Library,Command | Description$ The chosen equipment perpetually gains equip {1}.
+SVar:DBCleanup:DB$ Cleanup | ClearChosenCard$ True | ClearRemembered$ True
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/kembas_outfitter.txt:11]: Why ClearRemembered?

File: forge-gui/res/cardsfolder/upcoming/ichor_aberration.txt (hunk 1 above)
Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/ichor_aberration.txt]: Hmn, I'll make an issue for later as it might be more involved than I initially thought

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/ichor_aberration.txt]: Feel free to use the previous version, it's not a high priority

File: forge-gui/res/cardsfolder/upcoming/ghalma_the_shaper.txt [hunk 3]
```diff
@@ -0,0 +1,12 @@
+Name:Ghalma, the Shaper
+ManaCost:1 W W
+Types:Legendary Creature Elephant Artificer
+PT:3/3
+K:Vigilance
+T:Mode$ Attacks | ValidCard$ Card.Self | TriggerZones$ Battlefield | Execute$ TrigConjure | TriggerDescription$ Whenever CARDNAME attacks, conjure a card named Tempered Steel card into your hand, then create a 1/1 colorless Myr artifact creature token.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/ghalma_the_shaper.txt:6]: Get rid of extra …Card

**Review by tool4ever — APPROVED** [review]

File: forge-gui/res/cardsfolder/upcoming/hexgold_sledge.txt [hunk 4]
```diff
@@ -0,0 +1,10 @@
+Name:Hexgold Sledge
+ManaCost:2 R
+Types:Artifact Equipment
+K:For Mirrodin
+K:Equip:1
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigConjure | TriggerDescription$ When CARDNAME enters the battlefield conjure a card named Goblin Gaveleer onto the battlefield.
```
Comment (paulsnoops) [inline forge-gui/res/cardsfolder/upcoming/hexgold_sledge.txt:6]: enters the battlefield, conjure

On Oracle line too

File: forge-gui/res/cardsfolder/upcoming/ichor_aberration.txt [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+SVar:TrigPerpetualBuff:DB$ Effect | StaticAbilities$ PerpetualPump | RememberObjects$ Self | Duration$ Permanent | Name$ Ichor Aberration's Perpetual Effect | SubAbility$ ReanimateSelf
+SVar:PerpetualPump:Mode$ Continuous | Affected$ Card.IsRemembered | AddPower$ 1 | AddToughness$ 1 | EffectZone$ Command | AffectedZone$ Battlefield,Hand,Graveyard,Exile,Stack,Library,Command | Description$ This creature perpetually gets +1/+1.
+SVar:ReanimateSelf:DB$ Effect | StaticAbilities$ MayPlay | RememberObjects$ Self
+SVar:MayPlay:Mode$ Continuous | EffectZone$ Command | Affected$ Card.IsRemembered | MayPlay$ True | AffectedZone$ Graveyard | Description$ You may cast EFFECTSOURCE from your graveyard this turn.
+DeckHints:Ability$Proliferate
+DeckHas:Ability$Graveyard
+Oracle:Flying, defender\nAs long as Ichor Aberration's power is 7 or greater, it can attack as though it didn't have defender. \nWhenever you proliferate, if Ichor Aberation is in your graveyard or on the battlefield, Ichor Aberration perpetually gets +1/+1. You may cast it from your graveyard this turn.
```
Comment (paulsnoops) [inline forge-gui/res/cardsfolder/upcoming/ichor_aberration.txt:16]: defender. \nWhenever

No space here

File: forge-gui/res/cardsfolder/upcoming/norns_fetchling.txt [hunk 6]
```diff
@@ -0,0 +1,13 @@
+Name:Norn's Fetchling
+ManaCost:1 W
+Types:Creature Phyrexian Beast
+PT:1/1
+K:Toxic:1
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigBranch | TriggerDescription$ When CARDNAME enters the battlefield, conjure a card named Plains into your hand. If an opponent has three or more poison counters, you may seek a nonland card instead.
```
Comment (paulsnoops) [inline forge-gui/res/cardsfolder/upcoming/norns_fetchling.txt:6]: Does this need PrecostDesc$ Corrupted — ?

Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/norns_fetchling.txt:6]: I think this only works on activated abilities with a cost

**Review by paulsnoops — APPROVED** [review]

**Pull request merged**